fades/main.py

```diff
@@ -260,6 +260,9 @@ def go(argv):
     # run forest run!!
     python_exe = 'ipython' if args.ipython else 'python'
     python_exe = os.path.join(venv_data['env_bin_path'], python_exe)
+    # add the virtualenv /bin path to the child PATH.
+    os.environ['PATH'] = "{}:{}".format(venv_data['env_bin_path'], os.environ.get('PATH'))
+    logger.debug("child PATH: %s", os.environ)
```

Comment:
The logger in this code is called "l" (feel free to use this branch to enhance this name), not logger (note that logger is the module we imported).

Also, please start the logging line in uppercase.

Finally, don't log the WHOLE os.environ, just the PATH value